app/src/main/res/layout/fragment_subscription_details.xml

```diff
@@ -0,0 +1,258 @@
+          android:layout_marginTop="32dp"
+          android:layout_marginEnd="32dp"
+          android:background="#eaeaea"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@id/status"
+          />
+
+
+      <LinearLayout
+          android:id="@+id/content"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="32dp"
+          android:layout_marginTop="@dimen/big_margin"
+          android:layout_marginEnd="32dp"
+          android:orientation="vertical"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@id/top_separator"
+          >
+
+        <include
+            android:id="@+id/layout_active_subscription_content"
+            layout="@layout/layout_active_subscription_content"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            />
+
+        <include
+            android:id="@+id/layout_expired_subscription_content"
+            layout="@layout/layout_expired_subscription_content"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:visibility="gone"
+            />
+      </LinearLayout>
+
+      <View
+          android:id="@+id/bottom_separator"
+          android:layout_width="match_parent"
+          android:layout_height="1dp"
+          android:layout_marginStart="32dp"
+          android:layout_marginTop="32dp"
+          android:layout_marginEnd="32dp"
+          android:background="#eaeaea"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@id/content"
+          />
+
+
+      <ImageView
+          android:id="@id/info"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="@dimen/big_margin"
+          android:layout_marginTop="32dp"
+          android:layout_marginBottom="32dp"
+          app:layout_constraintBottom_toBottomOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@+id/bottom_separator"
+          app:layout_constraintVertical_bias="1"
+          app:srcCompat="@drawable/ic_info_grey"
+          />
+
+      <TextView
+          android:id="@+id/info_text"
+          style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="8dp"
+          android:layout_marginEnd="@dimen/big_margin"
+          android:text="APPC value can be volatile, the value shown may not match  the next  payment"
```

Comment:
add visibility gone here as this layout shows up before the skeleton and then disappears